tests/Tests/Mapping/PropertyWalkerTests .cs

```diff
@@ -0,0 +1,77 @@
+using System;
```

Comment:
This file needs a license header like so:
```suggestion
/* SPDX-License-Identifier: Apache-2.0
*
* The OpenSearch Contributors require contributions made to
* this file be licensed under the Apache-2.0 license or a
* compatible open source license.
*/
using System;
```

Reply:
Fixed. Anything more I need to change?